assets/js/blocks/active-filters/utils.js

```diff
@@ -5,40 +5,42 @@ import { __, sprintf } from '@wordpress/i18n';
 import { formatPrice } from '@woocommerce/base-utils';
 
 export const formatPriceRange = ( minPrice, maxPrice ) => {
-	let priceString;
-
 	if ( Number.isFinite( minPrice ) && Number.isFinite( maxPrice ) ) {
 		/* translators: %s min price, %s max price */
-		priceString = sprintf(
+		return sprintf(
 			__( 'Between %s and %s', 'woo-gutenberg-products-block' ),
 			formatPrice( minPrice ),
 			formatPrice( maxPrice )
 		);
-	} else if ( Number.isFinite( minPrice ) ) {
+	}
+
+	if ( Number.isFinite( minPrice ) ) {
 		/* translators: %s min price */
-		priceString = sprintf(
+		return sprintf(
 			__( 'From %s', 'woo-gutenberg-products-block' ),
 			formatPrice( minPrice )
 		);
-	} else {
-		/* translators: %s max price */
-		priceString = sprintf(
-			__( 'Up to %s', 'woo-gutenberg-products-block' ),
-			formatPrice( maxPrice )
-		);
 	}
 
-	return priceString;
+	/* translators: %s max price */
+	return sprintf(
+		__( 'Up to %s', 'woo-gutenberg-products-block' ),
+		formatPrice( maxPrice )
+	);
 };
 
 /**
- * Render item.
+ * Render a removable item in the active filters block list.
  * @param {string} type Type string.
  * @param {string} name Name string.
  * @param {*} removeCallback Callback to remove item.
  * @param {*} removeIcon Icon for the remove button.
```

Comment:
- removeCallback is a `Function`
- `removeIcon` doesn't appear to be an accepted argument anymore?